examples/hello_services/android/app/build.gradle

```diff
@@ -2,8 +2,8 @@ apply plugin: 'com.android.application'
 apply plugin: 'flutter'
 
 android {
-    compileSdkVersion 22
-    buildToolsVersion '22.0.1'
+    compileSdkVersion 25
+    buildToolsVersion '24.0.2'
```

Comment:
For a recent default installation of Android Studio, I see `buildToolsVersion "25.0.2"`. Any reason to stick with 24 here?